src/app/allocation/page.tsx

```diff
@@ -62,7 +63,12 @@ const NoKYC = ({ fullAddress }: { fullAddress: string }) => {
       </p>
       <FractalIDLogo className="h-[27px] w-[120px] sm:h-[44px] sm:w-[200px]" />
       <KYCButton address={fullAddress} />
-      <EligibilityFAQLink />
+      <Link
```

Comment:
Do we need the full next router for a local hash link?

Reply:
This is not the router it's the next [link component](https://nextjs.org/docs/pages/api-reference/components/link). I see no issues using it here but please let me know if there is one?